client/client.go

```diff
@@ -240,6 +240,9 @@ func (proxy *Proxy) Toxics() (Toxics, error) {
 // See https://github.com/Shopify/toxiproxy#toxics for a list of all Toxic types.
 func (proxy *Proxy) AddToxic(name, typeName, stream string, toxicity float32, attrs Attributes) (*Toxic, error) {
 	toxic := Toxic{name, typeName, stream, toxicity, attrs}
+	if toxic.Toxicity == -1 {
```

Comment:
Well.. it isn't. I put it there just so -1 can be used as the default anywhere and to be consistent. Maybe that's not something we care about though.